.github/workflows/run-on-pr.yaml

```diff
@@ -49,3 +49,20 @@ jobs:
 
       - name: Run tests
         run: tox -e github-${{ matrix.build-type }} -- -q --nomemory ${{ matrix.pytest-args }}
+  run-test-arm64:
+    name: Testing arm64-python${{ matrix.python-version }}-${{ matrix.build-type }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.8"
+        build-type:
+          - "nocext"
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          docker run --rm --privileged multiarch/qemu-user-static:register --reset
+      - uses: docker://multiarch/ubuntu-core:arm64-bionic
```

Comment:
Can't we use the arm python images https://hub.docker.com/r/arm64v8/python so we avoid having to download and install python every time?

Reply:
Thanks for the feedback. I have checked with arm64v8/python images, we can't give a dynamic tag to image like `docker://arm64v8/python:${{ manylinux.python-version }}` as it is giving a syntax error. So, even if we use `docker://arm64v8/python:3.8-buster` then we need to install python 3.6 version as it is not available by default. Please let me know if I am missing anything here.